syntax-highlighting-code-block.php

```diff
@@ -104,16 +106,25 @@ function enqueue_editor_assets() {
  * Asset(s) will only be enqueued if needed.
  */
 function register_frontend_assets() {
-	/**
-	 * Filters the style used for the code syntax block.
-	 *
-	 * The string returned must correspond to the filenames found at <https://github.com/scrivo/highlight.php/tree/master/styles>,
-	 * minus the file extension.
-	 *
-	 * @since 1.0.0
-	 * @param string $style Style.
-	 */
-	$style = apply_filters( 'syntax_highlighting_code_block_style', 'default' );
+	if ( has_filter( BLOCK_STYLE_FILTER ) ) {
+		/**
+		 * Filters the style used for the code syntax block.
+		 *
+		 * The string returned must correspond to the filenames found at <https://github.com/scrivo/highlight.php/tree/master/styles>,
+		 * minus the file extension.
+		 *
+		 * @since 1.0.0
+		 * @param string $style Style.
+		 */
+		$style = apply_filters( BLOCK_STYLE_FILTER, 'default' );
+	} else {
+		$style = get_option(
```

Comment:
As opposed to getting the option directly, how about rather creating a namespaced function like:

```php
function get_options() {
    $options = get_option( OPTION_NAME, [] );
    return array_merge(
        [
            'theme_name' => 'default',
        ],
        $options
    );
}

function get_option( $option_name ) {
    return get_options()[ $option_name ];
}
```

Then you can use `get_option( 'theme_name' )` down below.